assets/js/components/text-toolbar-button/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function TextToolbarButton( props ) {
```

Comment:
Can we deconstruct this to keep with the global coding style?
```suggestion
function TextToolbarButton( { className, ...props } ) {
```

Reply:
Thanks, that works for me :)